tests/vmi_configuration_test.go

```diff
@@ -2306,7 +2307,7 @@ var _ = Describe("Configurations", func() {
 			By("Check value in VM with dmidecode")
 			// Check on the VM, if expected values are there with dmidecode
 			res, err := expecter.ExpectBatch([]expect.Batcher{
-				&expect.BSnd{S: "[ $(sudo dmidecode -s chassis-asset-tag | tr -s ' ') -eq Test-123 ] && echo 'pass' || echo 'fail'\n"},
+				&expect.BSnd{S: "[ $(sudo dmidecode -s chassis-asset-tag | tr -s ' ') = Test-123 ] && echo 'pa''ss'\n"},
```

Comment:
other question , shouldnt it be `==` to the string ?
https://stackoverflow.com/questions/4277665/how-do-i-compare-two-string-variables-in-an-if-statement-in-bash

Reply:
I suggest you ref official docs and not QnA sites.
See https://linux.die.net/man/1/test

Reply:
`man [` (or `man test`) tells you the right operator is `=`, though they usually support `==` because it's a common "mistake"

Reply:
all good, the purpose i asked was because i wanted to hear the other opinions and way of thinking, thanks